docs/File-Formats.md

```diff
@@ -1310,3 +1311,56 @@ The cells contain treatment response values that can be postive and negative rea
 <tr><td>AEW541</td><td>Larotrectinib</td><td>TrkA/B/C inhibitor</td><td>https://en.wikipedia.org/wiki/Larotrectinib</td><td>>8</td><td>2.33</td><td>2.68</td></tr>
 <tr><td>AZD0530</td><td>Saracatinib</td><td>Src/Bcr-Abl inhibitor</td><td>https://en.wikipedia.org/wiki/Saracatinib</td><td>NA</td><td>>8</td><td>4.60</td></tr>
 </table>
+
+## Generic Assay
+Generic assay is a generic data type, which can holds different types of data on samples. cBioPortal supports any number of response data types imported from separate data files. Any meta information on generic assay are not part of the cBioPortal seed database, but are imported automatically from generic assay data files. For meta information already present in the database, all the fields are overwitten by subsequent imports of new generic assay data files. The `entity_stable_id` field must be unique within a data file.
+
+### Generic Assay meta file
+The meta file will be similar to meta files of other genetic profiles, such as mRNA expression. For generic assay data `LIMIT-VALUE` is used as `datatype`. The `LIMIT-VALUE` is validated to contain any continuous number optionally prefixed with a '>' or '<' threshold symbol (e.g., '>8.00'). For generic assay meta files values for `stable_id` are not pre-registered in the cBioPortal implementation, but can be freely chosen by the user. Requirement is that each generic assay meta file is assigned an unique `stable_id` by the user. This requirement is validated by the data validation scripts. 
+
+Required fields: 
+```
+cancer_study_identifier: Same value as specified in meta file of the study
+genetic_alteration_type: GENERIC_ASSAY
+generic_assay_type: the actual data type
+datatype: LIMIT-VALUE
+stable_id: Any unique identifier using a combination of alphanumeric characters, _ and -
+profile_name: A name describing the analysis.
+profile_description: A description of the data processing done.
+data_filename: <name of generic assay data file>
+show_profile_in_analysis_tab: true
+generic_entity_meta_properties: The meta properties you want to have in your meta data, you need to specify them here, properties must seperated by column (e.g., 'name,description')
```

Comment:
Just say "A comma separate list of generic entity properties".  See note below for more information.